app/assets/javascripts/notifications.js

```diff
@@ -183,15 +198,30 @@ var onReadyNotifications = function() {
     hideCategoriesMoods();
   });
 
+  $('#categories_moods').click(function() {
+    hideCategoriesMoods();
+  });
+
+  $('#categories_moods_text').click(function(event) {
+    event.stopPropagation();
+  });
+
   /* Tips */
-  $('.tip_notifications_button').click(function(){
+  $('.tip_notifications_button').click(function() {
     $(this).siblings('.tip_notifications').toggleClass("display_none");
     showBackdrop();
   });
 
-  $('.tip_close_notifications').click(function(){
-    $(this).closest('.tip_notifications').toggleClass("display_none");
-    hideBackdrop();
+  $('.tip_close_notifications').click(function() {
+    hideTipNotifications.call(this);
+  });
+
+  $('.tip_notifications').click(function() {
```

Comment:
It seems to me that we don't need this? Can you explain why we are closing this modal when we click it?
When we remove this, the corresponding test passes!